python/tools/Dockerfile.manylinux2014

```diff
@@ -0,0 +1,16 @@
+FROM quay.io/pypa/manylinux2014_x86_64
+
+ENV LD_LIBRARY_PATH=/usr/local/lib
+
+RUN yum update -y \
+    && yum install -y epel-release || yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-$(cut -d: -f5 /etc/system-release-cpe | cut -d. -f1).noarch.rpm \
```

Comment:
Is there no lsb_release equivalent to avoid having to do the manual output parsing via cut?

Reply:
there is a `/etc/redhat-release`.

But these lines are directly copied from the arrow installation instructions.